.goreleaser

```diff
@@ -0,0 +1,68 @@
+project_name: root-signing
+
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+  - COSIGN_EXPERIMENTAL=true
+
+# Prevents parallel builds from stepping on eachothers toes downloading modules
+before:
+  hooks:
+    - go mod tidy
+    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+
+gomod:
+  proxy: true
+
+sboms:
+  - artifacts: binary
+
+builds:
+  - id: root-verify
+    binary: root-verify-linux-{{ .Arch }}
+    no_unique_dist_dir: true
+    main: ./cmd/verify
+    goos:
+      - linux
+    goarch:
+      - amd64
+    goarm:
+      - 7
+    flags:
+      - -trimpath
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    ldflags:
+      - "{{ .Env.LDFLAGS }}"
+
+signs:
+  # Keyless
+  - id: keyless
+    signature: "${artifact}-keyless.sig"
+    certificate: "${artifact}-keyless.pem"
+    cmd: cosign
+    args: ["sign-blob", "--output-signature", "${artifact}-keyless.sig", "--output-certificate", "${artifact}-keyless.pem", "${artifact}"]
+    artifacts: binary
+  - id: checksum-keyless
+    signature: "${artifact}-keyless.sig"
+    certificate: "${artifact}-keyless.pem"
+    cmd: cosign
+    args: ["sign-blob", "--output-signature", "${artifact}-keyless.sig", "--output-certificate", "${artifact}-keyless.pem", "${artifact}"]
+    artifacts: checksum
+
+archives:
+  - format: binary
+    name_template: "{{ .Binary }}"
+    allow_different_binary_count: true
```

Comment:
I'm not very familiar with Goreleases, do we need this option?

Reply:
it is ok to have it

Reply:
aaah right, we don't need this anymore since we only have one os/arch. i'll remove this, and i'll add back in if it needs to be expanded.